ReGuilded/inject/index.js

```diff
@@ -1,33 +1,39 @@
 const { exec } = require("child_process");
 const inject = require("./inject.js");
+const log4js = require('log4js')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ReGuilded/ReGuilded/pullRequest?prid=7536470)
```suggestion
const log4js = require("log4js")
```